lib/system/nuttx/init.c

```diff
@@ -19,6 +19,7 @@ int metal_sys_init(const struct metal_init_params *params)
 {
 	int ret = metal_cntr_irq_init();
 
+	metal_unused(params);
```

Comment:
Nitpicking: To respect the coding rules used in other source code, consider declaring metal_unused as a variable at the top of the function (to apply in other functions as well).

Here it becomes:
```
int metal_sys_init(const struct metal_init_params *params)
{
	int ret = metal_cntr_irq_init();
	metal_unused(params);
	
	if (ret >= 0)
```

Reply:
Ok, done.